plugins/catalog-react/src/filters.ts

```diff
@@ -108,6 +108,16 @@ export class EntityTextFilter implements EntityFilter {
     return true;
   }
 
+  getBackendRequestParameter() {
+    return {
+      fullTextFilter: {
+        term: this.value,
+        // Update this to be more dynamic based on table columns.
+        fields: ['metadata.name', 'metadata.title'],
```

Comment:
Without this override, the search will default to the orderBy field only which breaks the starred entities widget.

Reply:
Worth adding `spec.profile.displayName` here, since it's a common display field (the presentation api does use it for example, where applicable(

Reply:
Added in https://github.com/backstage/backstage/pull/22131/commits/f845dd6f4fb72be3ada9f43615edc75cf9b59d20